src/selectbox_abstract.coffee

```diff
@@ -164,6 +166,16 @@ class SelectBoxAbstract
             $item.attr 'title', $option.attr 'title'
             $item.attr 'value', $option.val()
 
+            if @options.showcheckbox and @orig.is '[multiple]'
+                checkboxOptions =
+                    type  : 'checkbox'
+                    class : 'reform-checkbox'
+
+                checkboxOptions.checked = 'checked'  if $option.is ':selected'
+                checkboxOptions.checked = 'disabled' if $option.is ':disabled'
+
+                $item.append($('<input>', checkboxOptions))
```

Comment:
@gregorpraprotnik Found one mistake should be .disabled = 'disabled'. Will change it tomorrow.